easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb

```
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'phyx'
+version = '1.3'
+
+homepage = 'https://github.com/FePhyFoFum/%(name)s'
+description = "phyx performs phylogenetics analyses on trees and sequences."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+github_account = 'FePhyFoFum'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-1.01_link_openblas.patch']
+checksums = [
+    'b09677f98f997838b39cd2724463be06d0627ddc90a7e659dc90d4e02db624eb',  # v1.3.tar.gz
+    '8c43454c7ad87cfa9ac1eacef31fb651637f8ae218d6337beabbc98899eb574d',  # phyx-1.01_link_openblas.patch
+]
+
+builddependencies = [
+    ('Autotools', '20200321', '', ('GCCcore', '10.2.0')),
```

adding a subtoolchain is not necessary, EB is smart enough to know this.
```suggestion
    ('Autotools', '20200321'),
```